lms/services/lti_grading/_v13.py

```diff
@@ -66,7 +66,7 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
                 scopes=self.LTIA_SCOPES,
                 json=payload,
                 headers={"Content-Type": "application/vnd.ims.lis.v1.score+json"},
-            ).json()
```

Comment:
The change was made in: 

https://github.com/hypothesis/lms/pull/5098/files#diff-be7b2bca898d10b3182038f206d1e6adcbd52d567d965d682fe85f3588f9f3e9

which looked like a good idea at the time